client/components/Modal/PageCreateModal.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useState, FC } from 'react'
+import { useTranslation, Trans } from 'react-i18next'
+import { Button, Form, Input, Label, Modal, ModalBody, ModalHeader } from 'reactstrap'
+import moment from 'moment'
+import styled from 'styled-components'
+
+import { dark, gray, light } from '../../constants/colors'
+import Crowi from 'client/util/Crowi'
+import Icon from '../Common/Icon'
+
+const parentPath = (path: string) => {
+  if (path === '/') {
+    return path
+  }
+
+  if (path.match(/.+\/$/)) {
+    return path
+  }
+
+  return path + '/'
+}
+
+const NewIcon = styled(Icon)`
+  margin-right: 3px;
+`
+
+const TodayInputBox = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 8px;
+  width: 100%;
+`
+
+const BaseInput = styled(Input)`
+  background-color: ${light};
+  border: none;
+  border-bottom: 1px dotted ${dark};
+  border-radius: 0;
+  font-size: 0.9rem;
+  padding: 6px;
+  &:focus {
+    background-color: ${gray[200]};
+    border-color: #7faaaf;
+    box-shadow: 0 0 0 0.2rem rgba(67, 103, 107, 0.25);
+  }
+`
+
+const NewButton = styled(Button)`
+  height: 38px;
+`
+
+const CreateButton = styled(Button)`
+  margin: 0 0 0 auto;
+  height: 35px;
+`
+
+const FormLabel = styled(Label)`
+  font-size: 18px;
+  width: 100%;
+`
+
+const Path = styled.span`
+  font-size: 0.9rem;
+  padding: 6px;
+`
+
+const PortalNameInput = styled(BaseInput)`
+  max-width: 60px;
+`
+
+const PageNameInput = styled(BaseInput)`
+  flex: 1;
+  min-width: 0;
+`
+
+const CurrentPath = styled.code`
+  word-break: break-all;
+`
+
+const UnderTreePathInput = styled(BaseInput)`
+  flex: 1;
+  margin-right: 8px;
+`
+
+interface Props {
+  crowi: Crowi
+}
+
+const PageCreateModal: FC<Props> = ({ crowi }) => {
+  const { user } = crowi
```

Comment:
```suggestion
  const user = crowi.getUser()
```

Please use instance method🙏